modules/market/src/main/java/com/opengamma/strata/market/curve/CurveMetadata.java

```diff
@@ -65,6 +65,15 @@
   public abstract Optional<DayCount> getDayCount();
 
   /**
+   * Gets the calibration information, optional.
+   * <p>
+   * When a curve is calibrated, useful information is obtained, which can be stored here.
+   * 
+   * @return the calibration information
+   */
+  public abstract Optional<CurveCalibrationInfo> getCalibrationInfo();
```

Comment:
Why is this on the metadata rather than the curve itself? It seems a bit of a stretch to describe the Jacobian matrix as metadata. Arguably it's part of the curve data.